helpdesk_mgmt_project/migrations/16.0.2.2.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Create a column with the old data and we fill it in."""
+    openupgrade.logged_query(
```

Comment:
Use `copy_columns` instead.

Reply:
Anyway, I think you don't need to copy it. Just rename it, as now data goes to `ir_property` table.

Reply:
You can use `rename_columns` openupgradelib method.